### ISSUE TITLE
`docker-machine start` fails on Mac OSX 10.10.5

### ISSUE THREAD

**Issue by Supermighty** [docker/machine#1723]
Starting a docker machine fails. It looks like a VirtualBox error. I was able to find the error, and solution, on this [virtualbox issue ticket](https://www.virtualbox.org/ticket/14412). The solution was to install  a VirtualBox TestBuild.

From  VBox.log
```
VirtualBox VM 5.0.2 r102096 darwin.amd64 (Aug 13 2015 18:38:03) release log
00:00:00.020443 Log opened 2015-08-18T13:12:36.637753000Z
00:00:00.020443 Build Type: release
00:00:00.020454 OS Product: Darwin
00:00:00.020459 OS Release: 14.5.0
00:00:00.020464 OS Version: Darwin Kernel Version 14.5.0: Wed Jul 29 02:26:53 PDT 2015; root:xnu-2782.40.9~1/RELEASE_X86_64
00:00:00.020547 DMI Product Name: MacBookPro12,1
00:00:00.020588 DMI Product Version: 1.0
00:00:00.020594 Host RAM: 8192MB total, 3388MB available
00:00:00.020596 Executable: /Applications/VirtualBox.app/Contents/MacOS/VBoxHeadless
00:00:00.020596 Process ID: 631
00:00:00.020597 Package type: DARWIN_64BITS_GENERIC
00:00:00.021787 Installed Extension Packs:
00:00:00.021796   None installed!
00:00:00.022561 Console: Machine state changed to 'Starting'
00:00:00.028158 SUP: Loading failed for VMMR0.r0 (/Applications/VirtualBox.app/Contents/MacOS/VMMR0.r0) rc=VERR_VMM_SMAP_BUT_AC_CLEAR
00:00:00.028201 PDMLdr: pdmR3LoadR0U: pszName="VMMR0.r0" rc=VERR_VMM_SMAP_BUT_AC_CLEAR szErr="SUPR3LoadModule: supLoadModule returned VERR_VMM_SMAP_BUT_AC_CLEAR"
00:00:00.028209 VMSetError: /Users/vbox/tinderbox/mac-rel/src/VBox/VMM/VMMR3/VM.cpp(595) int vmR3CreateU(UVM*, uint32_t, int (*)(UVM*, VM*, void*), void*); rc=VERR_VMM_SMAP_BUT_AC_CLEAR
00:00:00.028214 VMSetError: Failed to load VMMR0.r0
00:00:00.028299 ERROR [COM]: aRC=NS_ERROR_FAILURE (0x80004005) aIID={872da645-4a9b-1727-bee2-5585105b9eed} aComponent={ConsoleWrap} aText={Failed to load VMMR0.r0 (VERR_VMM_SMAP_BUT_AC_CLEAR)}, preserve=false aResultDetail=0
00:00:00.028374 Console: Machine state changed to 'PoweredOff'
00:00:00.030367 Power up failed (vrc=VERR_VMM_SMAP_BUT_AC_CLEAR, rc=NS_ERROR_FAILURE (0X80004005))
```

/var/log/system.log
```
kernel[0]: vboxdrv: Bad CPU context error at line 4886 in SUPDrv.cpp: EFLAGS.AC is 0!
```

**Comment by nathanleclaire** [comment]
Argh, rough.  Thanks for filing the ticket to let us know!

**Comment by ehazlett** [comment]
Thanks for the feedback!

**Comment by dgageot** [comment]
Closing. This was fixed in VBox 5.0.4
